modules/ztp-image-based-upgrade-generate-seed-image.adoc

```diff
@@ -0,0 +1,207 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-seed-generation_{context}"]
+= Generating a seed image with the {lcao}
+
+Use the {lcao} to generate the seed image with the `SeedGenerator` CR. The Operator does the following tasks:
+
+// seed cluster, some things are cluster specific that are not included
+// all operator, LCAO, OADP, LVMS, included in the seed image, telco day2 stuff is incl
+// system config checks ()
+
+* Checks for required system configurations
+* Performs any necessary system cleanup before generating the seed image
+* Launches the image generation, which involves the following tasks:
+** Stops cluster operators
+** Prepares the seed image configuration
+** Generates and pushes the seed image to the image repository specified in the `ImageBasedUpgrade` CR
+** Restores cluster operators
+** Expires seed cluster certificates
+* Generates new certificates for the seed cluster
+* Restores and updates the `SeedGenerator` CR on the seed cluster, and the `ManagedCluster` CR on the hub cluster
+
+[IMPORTANT]
+====
+The generated seed image does not include any site-specific data.
+====
+
+//TODO Expand on what the seed image contains and explicitly say what it doesn't
+
+.Prerequisites
+
+* Deploy a {sno} cluster with a DU profile.
+* Install the {lcao} on the seed cluster.
+* Install the OADP Operator on the seed cluster.
+* Log in as a user with `cluster-admin` privileges.
+* The seed cluster has the same CPU topology as the target cluster.
+* If using {rh-rhacm}, the seed cluster is registered as a managed cluster.
+* The {lcao} deployed on the target cluster is compatible with the version in the seed image.
+// Are there more prereqs?
+
+.Procedure
+
+. (Optional) If you use {rh-rhacm} to manage your clusters, detach the seed cluster from the hub cluster either manually or if using ZTP, by removing the `SiteConfig` CR from the `kustomization.yaml`.
```

Comment:
@alosadagrande @browsell
Is this still required?

Reply:
Brent told me that hubKubeconfig should not be in the customer docs, these two options are needed instead